Cesium.Runtime/CPtr.cs

```diff
@@ -19,7 +19,7 @@ private CPtr(long value)
 
 /// <summary>A class encapsulating an object pointer.</summary>
 /// <typeparam name="T">Type this pointer may be resolved to.</typeparam>
-public unsafe readonly struct CPtr<T> where T : unmanaged
+public readonly unsafe struct CPtr<T> where T : unmanaged
```

Comment:
And maybe `VoidPtr`. I do like CPtr, but it sounds too broad for me. But that's just remark. I do have only vague feeling here.